examples/data-sources/auth0_connection_scim_configuration/data-source.tf

```diff
@@ -0,0 +1,26 @@
+resource "auth0_connection" "my_enterprise_connection" {
+  name         = "my-enterprise-connection"
+  display_name = "My Enterprise Connection"
+  strategy     = "okta"
+
+  options {
+    client_id              = "1234567"
+    client_secret          = "1234567"
+    issuer                 = "https://example.okta.com"
+    jwks_uri               = "https://example.okta.com/oauth2/v1/keys"
+    token_endpoint         = "https://example.okta.com/oauth2/v1/token"
+    authorization_endpoint = "https://example.okta.com/oauth2/v1/authorize"
+  }
+}
+
+resource "auth0_connection_scim_configuration" "my_conn_scim_configuration" {
+  depends_on    = [auth0_connection.my_enterprise_connection]
+  connection_id = auth0_connection.my_enterprise_connection.id
+}
+
+# A data source for an Auth0 Connection SCIM Configuration.
+data "auth0_connection_scim_configuration" "my_conn_scim_configuration_data" {
+  depends_on    = [auth0_connection_scim_configuration.my_conn_scim_configuration]
```

Comment:
We don't need the depends_on here, as it's implied in the `connection_id` section below.